timemachine/cpp/src/kernels/k_nonbonded_pair_list.cuh

```diff
@@ -156,6 +158,9 @@ void __global__ k_nonbonded_pair_list(
         g_qi += FLOAT_TO_FIXED_DU_DP<RealType, FIXED_EXPONENT_DU_DCHARGE>(charge_scale * qj * inv_dij * ebd);
         g_qj += FLOAT_TO_FIXED_DU_DP<RealType, FIXED_EXPONENT_DU_DCHARGE>(charge_scale * qi * inv_dij * ebd);
 
+        g_wi += FLOAT_TO_FIXED_NONBONDED(delta_prefactor * delta_w);
```

Comment:
Any reason not to move this into the `if (du_dp)` block?

Reply:
Not that I see; this was just keeping with the convention used for the other parameter gradients here (accumulate into the register outside the `if (du_dp)` block; accumulate using `atomicAdd` inside the block)